runelite-client/src/main/java/net/runelite/client/plugins/implings/ImplingsConfig.java

```diff
@@ -36,15 +36,22 @@
 @ConfigGroup("implings")
 public interface ImplingsConfig extends Config
 {
+	enum ImplingMode
```

Comment:
Does this work? I think it makes the enum class package private, and the config stuff needs to be able to access it.

Reply:
It does yeah, I wasn't sure whether to put this in the plugin class or in the config. What would you suggest?

Reply:
I must be mis-remembering then. I will check it later. This is probably fine as it is.